src/main.rs

```diff
@@ -578,6 +580,9 @@ async fn main() -> Result<(), Error> {
                                     warn!("No valid crates link");
                                 }
                                 return Err(format_err!("Not high enough metrics ({:?} stars < {}, and {:?} cargo downloads < {}): {}", github_stars, required_stars, cargo_downloads, MINIMUM_CARGO_DOWNLOADS, list_item));
+                                if !ITEM_REGEX.is_match(&list_item) {
+                                    return Err(format_err!("Item does not match the template: {}. See https://github.com/rust-unofficial/awesome-rust/blob/main/CONTRIBUTING.md#tldr", list_item));
+                                }
```

Comment:
This is code after a `return` which the compiler is flagging as "unreachable expression"

Reply:
Indeed it is. My apologies. I was testing this with some of the code commented out so as not to trigger the throttling issues, and uncommented things wrong at the last minute before pushing. Will fix and retest.

Reply:
This is fixed